### PR TITLE
Add owner refs 

### PR DESCRIPTION
**Pull request by tantona**
*Issue #, if available:*
This PR adds ownerrefs for components applied to EKS clusters.  This ensures that when the cluster is delerted - all CRs for components deployed to this . cluster will get garbage collected.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by tantona** [comment]
I bumped the resources available to the controller manager bc I think it was getting OOM killed which was causing some connection issues :/

File: pkg/controller/clusterrole/clusterrole_controller.go [hunk 1]
```diff
@@ -139,6 +139,23 @@ func (r *ReconcileClusterRole) Reconcile(request reconcile.Request) (reconcile.R
...
 
+	if len(instance.ObjectMeta.OwnerReferences) < 1 {
+		// Add Owner Reference
+		instance.ObjectMeta.OwnerReferences = []metav1.OwnerReference{
+			{
+				APIVersion:         "cluster.eks.amazonaws.com/v1alpha1",
+				Kind:               "EKS",
+				Name:               cluster.ObjectMeta.Name,
+				UID:                cluster.ObjectMeta.UID,
+				Controller:         func(b bool) *bool { return &b }(true),
```
Comment (Magizhchi) [inline pkg/controller/clusterrole/clusterrole_controller.go]: Question: what is the controller tag used here for ?

**Comment by Magizhchi** [comment]
LGTM

File: pkg/controller/eks/components.go [hunk 2]
```diff
@@ -58,6 +58,21 @@ func deleteComponents(ownerName, ownerNamespace string, c client.Client, logger
 				item := obj
 				delete = append(delete, &item)
 			}
+		case *componentsv1alpha1.ClusterRoleList:
```
Comment (MadVikingGod) [inline pkg/controller/eks/components.go]: These components need to be added to the list on L14

**Pull request merged**